### PR TITLE
[ws-manager] Reduce readiness probe initialDelaySeconds

### PR DESCRIPTION
**Pull request by aledbf**
## Release Notes
```release-note
[ws-manager] Reduce readiness probe initialDelaySeconds
```

### CODE REVIEW

**Comment by roboquat** [comment]
@aledbf: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: dev/loadgen/prod-benchmark.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## start with
 ##    loadgen benchmark prod-benchmark.yaml
 
-workspaces: 500
+workspaces: 60
```
Comment (sagor999) [inline dev/loadgen/prod-benchmark.yaml:4]: this is intentional? seems like it shouldn't be part of this PR?

Comment (aledbf) [reply dev/loadgen/prod-benchmark.yaml]: Yes, but I can change that in a different PR. Fixed

**Review by kylos101 — APPROVED** [review]
:+1:
